src/ship/PlayerShipController.cpp

```diff
@@ -166,6 +166,13 @@ REGISTER_INPUT_BINDING(PlayerShipController)
 	auto landingGroup = controlsPage->GetBindingGroup("LandingControl");
 	input->AddActionBinding("BindToggleLandingGear", landingGroup, Action({ SDLK_n }));
 	input->AddAxisBinding("BindControlLandingGear", landingGroup, Axis());
+
+	auto radarGroup = controlsPage->GetBindingGroup("RadarControl");
+	input->AddActionBinding("BindRadarZoomReset", radarGroup, Action({ SDLK_UP }));
+	input->AddActionBinding("BindRadarToggleMode", radarGroup, Action({ SDLK_DOWN }));
```

Comment:
Nitpick: I personally expect the "down" arrow key to be semantically associated with "reset" - it has both the "go back" semantic association and is in-line with the other manual zoom mode keys by default.

Reply:
Fixed in d0e849130043b3ccf6a84d15cc0c026a8e5e542c